### DISCUSSION TITLE
Caps lock hot key

### DISCUSSION THREAD

**Discussion by nicksolyom** [hrvach/deskhop#55] [Q&A] [answered]
Hello!
Is there a way to make caps lock toggle the actual caps lock function rather than switch devices? I want to type in all caps sometimes.

**Comment by hrvach** [comment, marked as answer]
Yes, check the src/user_config.h header file and choose the hotkey to your own liking. 

It defines:
#define HOTKEY_TOGGLE HID_KEY_CAPS_LOCK

Pick any one you want from:
https://github.com/hathach/tinyusb/blob/master/src/class/hid/hid.h

... rebuild and flash.
